docker/README.md

```diff
@@ -15,6 +15,10 @@ Supported os/architectures:
 
 You can start a local server on port 3000  with demo data with
 
+```
+docker pull cr.signalk.io/signalk/signalk-server
+```
```

Comment:
Let's inline the references, so that we consistently use `cr.signalk.io/signalk/signalk-server` (always include the registry hostname) in the readme. Then for example here we don't need the separate `pull` line and we can keep just the oneliner.